components/osc-host-application/ApplicationForm.js

```diff
@@ -192,11 +184,21 @@ const messages = defineMessages({
   },
   aboutYourCommunityTitle: {
     id: 'HostApplication.form.aboutYourCommunity',
-    defaultMessage: 'About your community {optional} {padlock}',
+    defaultMessage: 'About your community {optional}',
+  },
+  aboutYourCommunitySubtitle: {
+    id: 'HostApplication.form.aboutYourCommunity.subtitle',
+    defaultMessage:
+      'If applicable, please share information about your community and your events so we can properly consider your application.',
   },
   aboutYourCodeTitle: {
     id: 'HostApplication.form.code',
-    defaultMessage: 'About your Code {optional} {padlock}',
+    defaultMessage: 'About your Code {optional}',
```

Comment:
```suggestion
    defaultMessage: 'About your Code',
```

Reply:
confusing as this element is not optional if applying on behalf of a code-centred project :)

Reply:
Should we fix this as part of the issue instead? https://github.com/opencollective/opencollective/issues/6053